Observer/Category.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Metrilo\Analytics\Observer;
+
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Event\ObserverInterface;
+
+class Category implements ObserverInterface
+{
+    /**
+     * @param \Metrilo\Analytics\Helper\Data                     $helper
+     * @param \Metrilo\Analytics\Helper\ApiClient                $apiClient
+     * @param \Metrilo\Analytics\Helper\CustomerSerializer       $customerSerializer
+     * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+     */
+    public function __construct(
+        \Metrilo\Analytics\Helper\Data                                  $helper,
+        \Metrilo\Analytics\Helper\ApiClient                             $apiClient,
+        \Metrilo\Analytics\Helper\CategorySerializer                    $categorySerializer,
+        \Magento\Catalog\Model\ResourceModel\Category\CollectionFactory $categoryCollection,
+        \Magento\Framework\App\Config\ScopeConfigInterface              $scopeConfig
+    ) {
+        $this->helper             = $helper;
+        $this->apiClient          = $apiClient;
+        $this->categorySerializer = $categorySerializer;
+        $this->categoryCollection = $categoryCollection;
+        $this->scopeConfig        = $scopeConfig;
+    }
+    
+    private function mapConfigToStore($storeIds) {
```

Comment:
I still don't find this method name descriptive

Reply:
getStoreIdPerProject sounds reasonable to me.